authentication/src/main/kotlin/com/uber/sdk2/auth/internal/AuthActivity.kt

```diff
@@ -71,6 +71,12 @@ class AuthActivity : AppCompatActivity() {
 
   override fun onResume() {
     super.onResume()
+
+    if (!authStarted) {
+      init()
+      authStarted = true
```

Comment:
Do you have the right intent flags set to keep multiple activities from happening? Or can this become a state/lifecycle bug? Can onResume be hit twice?

Reply:
yes, AuthActivity has a `singleTask` launchMode set through the manifest and exported flag is set as false so it could only be started through the sdk